### ISSUE TITLE
build fails with latest version of @nestjs/common

### ISSUE THREAD

**Issue by yuristsepaniuk** [wbhob/nest-middlewares#11]
please check out thread https://github.com/nestjs/nest/issues/1204

@next-middlewares/express-session uses bad typings, looks like referenced interface was removed since nest 5.0.0

**Comment by wbhob** [comment]
I'd appreciate a PR if you have time, I'm really busy right now and don't have time to implement this.

**Comment by yuristsepaniuk** [comment]
@wbhob I would love too, and I have very good news for you, we don't need any PRs. index.ts is fine, there is no usages of ex code, but index.d.ts still uses it. I am stuck to build the project because of another issue, let me share it.

`packages/express-bearer-token/index.ts:12:41 - error TS2694: Namespace 'bearerToken' has no exported member 'Options'.`

So I got rid of express-bearer-token and tried to build the project again. And it succeeded. After that I checked index.d.ts again and it looks good. So it happened that build for old version just produced bad typings.

So your steps should be the following:
1. build the project again, I am not sure if you ok to skip express-bearer-token problem as me.
2. publish new version to npm.

Thx
-Yura

**Comment by yuristsepaniuk** [comment]
@wbhob please let me know once it is done, I will check my project and give you feedback if our issue is fixed.

**Comment by wbhob** [comment]
5.1.1 published, please give it a try and tell me if it's fixed!

**Comment by yuristsepaniuk** [comment]
@wbhob It works! Please close! Thank you!

**Comment by wbhob** [comment]
Thanks for reporting :) I almost forgot to publish
